person.rb

```diff
@@ -0,0 +1,38 @@
+require_relative './nameable'
+
+class Person < Nameable
+  attr_accessor :name, :age
```

Comment:
- Hey, we also need the `:books, :rentals, :parent_permission` to be added here based on the relationship they possess.